patches/server/0976-Prevent-sending-oversized-item-data-in-equipment-and.patch

```diff
@@ -48,26 +48,25 @@ index 0000000000000000000000000000000000000000..e9436f8a73ee0a02096d66e14d73edaa
 +        if (projectiles.isEmpty()) {
 +            return projectiles;
 +        }
-+        final List<ItemStack> items = projectiles.getItems();
-+        final List<ItemStack> sanitized = new ArrayList<>();
-+        for (int i = 0; i < Math.min(items.size(), 3); i++) {
-+            // we want to preserve item type as vanilla client can change visuals based on type
-+            sanitized.add(new ItemStack(items.get(i).getItemHolder()));
-+        }
-+        return ChargedProjectiles.of(sanitized);
++
++        return ChargedProjectiles.of(List.of(
++            new ItemStack(projectiles.contains(Items.FIREWORK_ROCKET) ? Items.FIREWORK_ROCKET : Items.ARROW)
```

Comment:
Does the display no longer change with multiple in the list?

Reply:
I'm not sure if this was ever the case at least not with the default resource pack. But if so the whole things should be removed or a fake item added i guess.